README.md

```diff
@@ -1,548 +1,309 @@
-        name: "ERC20LRTST",
-        symbol: "ETHIcon",
-        imageUri: "ETHIcon",
-        decimals: 18,
-        resourceId:
-          "0x0000000000000000000000000000000000000000000000000000000000000300",
-        feeSettings: {
-          type: "basic",
-          address: "0x78E5b9cEC9aEA29071f070C8cC561F692B3511A6",
-        },
-      },
-    ],
-  },
-];
+For substrate usage you need to install polkadotjs packages
+```bash
+yarn add @polkadot/api @polkadot/extension-dapp @polkadot/util-crypto
 ```
 
-We are going to use the SDK with `NodeJS` so, we are going to use the bridge admin to make the transfer. For this instantiate a wallet"
-
-```ts
-import { ethers } from 'ethers'
-import { NonceManager } from "@ethersproject/experimental";
+## Usage
 
-// connect to one of the nodes
-const provider = new ethers.providers.JsonRpcProvider('http://localhost:8545');
+### EVM
+#### ERC20 and ERC721 approvals
 
-// instantiate the waller with the following mnemonic
-const wallet = ethers.Wallet.fromMnemonic(
-  "black toward wish jar twin produce remember fluid always confirm bacon slush",
-  "m/44'/60'/0'/0/0",
+First you need to ensure that erc20 is approved for the erc20 hadler and fee handler contract. You can do it on youre own or use our function for it.
+```typescript
+import { ERC20__factory } from '@buildwithsygma/sygma-contracts'
+import { approve } from '@buildwithsygma/sygma-sdk-core/EVM'
+const erc20Instance = ERC20__factory.connect("0x1234567890123456789012345678901234567890", signer) // ethers signer
+// getting gas price
+const gasPrice = await isEIP1559MaxFeePerGas(
+  signer.provider as providers.Web3Provider
 );
-
-// connect the wallet to the provider
-const walletConnected = wallet.connect(provider);
-
-// use the NonceManager to avoid issues with the nonce
-const managedSigner = new NonceManager(walletConnected);
-```
-
-Then we create a `setup` object to pass to the `Sygma` class. This setup uses the `bridgeSetup` defined above with the contract addresses.
-
-```ts
-const setup = { bridgeSetup }
-
-const chainbridge = new Sygma(setup)
+const receipt = await approve(
+  "100", // amount
+  erc20Instance, // erc20 instance
+  "0x0987654321098765432109876543210987654321", // erc20 handler address
+  1, // number of confirations
+  {
+    gasPrice, // gas price
+  }
+);
+// same should be for fee handler if you collect fee as erc20 token
+// we will look at it in the next step
 ```
-
-Now we are ready to initialize connection:
-
-```ts
-const bridgeEvents = sygma.initializeConnectionRPC(wallet.address)
+#### Fee calculation
+The next step is to calculate fee data:
+
+```typescript
+// import function depending on your fee strategy
+import { calculateBasicfee } from '@buildwithsygma/sygma-sdk-core/EVM'
+
+const feeData = await calculateBasicfee({
+  basicFeeHandlerAddress: '0x1234...',
+  provider: new ethers.providers.JsonRpcProvider('https://mainnet.infura.io/v3/YOUR-PROJECT-ID'),
```

Comment:
Here also should be convenient to add as commente `// also you can use valid alchemy API KEY`

Reply:
Done